monkey/infection_monkey/credential_collectors/mimikatz_collector/mimikatz_credential_collector.py

```diff
@@ -23,7 +24,11 @@ def _to_credentials(win_creds: Sequence[WindowsCredentials]) -> [Credentials]:
         for win_cred in win_creds:
             identities = []
             secrets = []
-            if win_cred.username:
+
+            # Mimikatz picks up users created by the Monkey even if they're successfully deleted
+            # since it picks up creds from the registry. The newly created users are not removed
+            # from the registry until a reboot of the system, hence this check.
```

Comment:
Have you checked? We need to make sure these users aren't left there forever

Reply:
Yes, checked